website/source/docs/upgrade-specific.html.md

```diff
@@ -14,6 +14,18 @@ details provided for their upgrades as a result of new features or changed
 behavior. This page is used to document those details separately from the
 standard upgrade flow.
 
+## Consul 1.1.0
+
+### Removal of deprecated features
+
+The following previously deprecated fields and config options have been removed:
+
+ - `CheckID` has been removed from config file check definitions (use `id` instead).
+ - `script` has been removed from config file check definitions (use `args` instead).
+ - `enableTagOverride` is no longer valid in service definitions (use `enable_tag_override` instead).
+ - The deprecated set of metric names (beginning with `consul.consul.`) has been removed
+ along with the `enable_deprecated_names` option from the metrics configuration.
+
```

Comment:
Should we link to the changelog/update guide that deprecated them in the first place to highlight how long people this surprises have had to notice?